### PR TITLE
Fixed label size issue

### PR DESCRIPTION
**Pull request by monika-Dianomic**

### CODE REVIEW

File: src/app/components/core/settings/settings.component.html [hunk 1]
```diff
@@ -223,15 +223,15 @@
...
-          <label class="label is-small">Developer Features</label>
+          <label class="label lbl-top">Developer Features</label>
           <div class="control">
             <input type="checkbox" (click)="setDeveloperFeatures($event.target.checked)"
               [checked]="developerFeaturesService.getDeveloperFeatureControl()">
           </div>
         </div>
         <div class="column is-2"></div>
         <div class="column is-3">
-          <label class="label is-small">Readings Graph Alpha Control</label>
```
Comment (praveen-garg) [inline src/app/components/core/settings/settings.component.html:234]: We should have kept as is

File: src/app/components/core/control-dispatcher/pipelines/add-pipeline/add-control-pipeline.component.html [hunk 2]
```diff
@@ -221,7 +221,7 @@
                             <div *ngIf="selectedDestinationType?.name && !(selectedDestinationType.name === 'Any' || selectedDestinationType.name === 'Broadcast')"
                               class="field is-horizontal">
                               <div class="field-label is-small">
-                                <label class="label is-small">Name</label>
+                                <label class="label lbl-child">Name</label>
```
Comment (praveen-garg) [inline src/app/components/core/control-dispatcher/pipelines/add-pipeline/add-control-pipeline.component.html:224]: this is okay;

File: src/app/components/core/control-dispatcher/api-flow/add-edit-api-flow/add-edit-api-flow.component.css [hunk 3]
```diff
@@ -12,12 +12,6 @@ table tr td {
     text-decoration: none;
 }
 
-.sub {
```
Comment (praveen-garg) [inline src/app/components/core/control-dispatcher/api-flow/add-edit-api-flow/add-edit-api-flow.component.css:15]: What was this for?

File: src/styles.css [hunk 4]
```diff
@@ -418,4 +413,17 @@ a.dropdown-item:hover {
   .desc-text{
     font-size: 0.95rem;
   }
+
+  .label, .is-small-font {
```
Comment (praveen-garg) [inline src/styles.css:417]: What is is small font?

File: src/styles.css [hunk 5]
```diff
@@ -418,4 +413,17 @@ a.dropdown-item:hover {
...
+    font-size: 0.9rem !important;
+    font-weight: 500;
+  }
+
+  .lbl-child {
+    font-size: 0.85rem !important;
+  }
+
+  .lbl-top {
```
Comment (praveen-garg) [inline src/styles.css:426]: This should not be named like this

label is-small should be 0.75rem

**Review by praveen-garg — CHANGES_REQUESTED** [review]
Just fix the regression

Use 
label
.lbl-child

File: src/app/components/core/control-dispatcher/api-flow/add-edit-api-flow/add-edit-api-flow.component.css (hunk 3 above)
Comment (monika-Dianomic) [reply src/app/components/core/control-dispatcher/api-flow/add-edit-api-flow/add-edit-api-flow.component.css:15]: I've checked. I didn't see any use of this class.

File: src/styles.css (hunk 4 above)
Comment (monika-Dianomic) [reply src/styles.css:417]: This class is being used in `configuration-group` component where we need same css property but that is not label. I can remove it from here and use it at component level.

File: src/app/components/core/control-dispatcher/add-control-acl/add-control-acl.component.html [hunk 6]
```diff
@@ -55,8 +55,8 @@
                           <div class="column service-control">
                             <div class="field is-horizontal">
                               <div class="field-label is-small">
-                                <label class="label" *ngIf="sharedService.checkAuth()">Names</label>
-                                <label class="label" *ngIf="!sharedService.checkAuth()">{{serviceNameList.length > 1 ?
+                                <label class="label lbl-child" *ngIf="sharedService.checkAuth()">Names</label>
```
Comment (praveen-garg) [inline src/app/components/core/control-dispatcher/add-control-acl/add-control-acl.component.html:58]: this also needs a class label?

File: src/app/components/core/control-dispatcher/add-control-acl/add-control-acl.component.html [hunk 7]
```diff
@@ -78,7 +78,7 @@
                             </div>
                             <div class="field is-horizontal">
                               <div class="field-label is-small">
-                                <label class="label" *ngIf="sharedService.checkAuth()">Types</label>
+                                <label class="label lbl-child" *ngIf="sharedService.checkAuth()">Types</label>
                                 <label class="label" *ngIf="!sharedService.checkAuth()">{{serviceTypeList.length > 1 ?
```
Comment (praveen-garg) [inline src/app/components/core/control-dispatcher/add-control-acl/add-control-acl.component.html:82]: Why do we need class label?

File: src/app/components/core/control-dispatcher/add-control-acl/add-control-acl.component.html [hunk 8]
```diff
@@ -122,7 +122,7 @@
                       <div class="column acl-control">
                         <div class="field is-horizontal">
                           <div class="field-label is-small">
-                            <label class="label">URL</label>
+                            <label class="label lbl-child">URL</label>
```
Comment (praveen-garg) [inline src/app/components/core/control-dispatcher/add-control-acl/add-control-acl.component.html:125]: As above

File: src/app/components/core/control-dispatcher/add-control-acl/add-control-acl.component.html [hunk 9]
```diff
@@ -138,7 +138,7 @@
                         </div>
                         <div class="field is-horizontal">
                           <div class="field-label is-small">
-                            <label class="label">ACLs</label>
+                            <label class="label lbl-child">ACLs</label>
```
Comment (praveen-garg) [inline src/app/components/core/control-dispatcher/add-control-acl/add-control-acl.component.html:141]: As above

File: src/app/components/core/control-dispatcher/add-control-script/add-step/add-configure/add-configure.component.html [hunk 10]
```diff
@@ -1,6 +1,6 @@
 <div class="field is-horizontal">
   <div class="field-label is-small">
-    <label class="label">Category</label>
+    <label class="label lbl-child">Category</label>
```
Comment (praveen-garg) [inline src/app/components/core/control-dispatcher/add-control-script/add-step/add-configure/add-configure.component.html:3]: As above

File: src/app/components/core/control-dispatcher/add-control-script/add-step/add-configure/add-configure.component.html [hunk 11]
```diff
@@ -37,7 +37,7 @@
 </div>
 <div *ngIf="config?.category" class="field is-horizontal">
   <div class="field-label is-small">
-    <label class="label">Config Item</label>
+    <label class="label lbl-child">Config Item</label>
```
Comment (praveen-garg) [inline src/app/components/core/control-dispatcher/add-control-script/add-step/add-configure/add-configure.component.html:40]: As above

File: src/app/components/core/control-dispatcher/add-control-acl/add-control-acl.component.html [hunk 12]
```diff
@@ -78,7 +78,7 @@
                             </div>
                             <div class="field is-horizontal">
                               <div class="field-label is-small">
-                                <label class="label" *ngIf="sharedService.checkAuth()">Types</label>
+                                <label class="label lbl-child" *ngIf="sharedService.checkAuth()">Types</label>
```
Comment (praveen-garg) [inline src/app/components/core/control-dispatcher/add-control-acl/add-control-acl.component.html:81]: We have check in next line as well

File: src/styles.css (hunk 4 above)
Comment (praveen-garg) [reply src/styles.css:417]: sure. and use label instead of .label? (tag vs class)

File: src/styles.css (hunk 5 above)
Comment (praveen-garg) [reply src/styles.css:426]: We can get rid of this is-small by using is-size-7?

File: src/app/components/core/control-dispatcher/add-control-script/add-step/add-script/add-script.component.html [hunk 13]
```diff
@@ -1,6 +1,6 @@
 <div class="field is-horizontal">
   <div class="field-label is-small">
-    <label class="label">Name</label>
+    <label class="label lbl-child">Name</label>
```
Comment (praveen-garg) [inline src/app/components/core/control-dispatcher/add-control-script/add-step/add-script/add-script.component.html:3]: this is a child label?

File: src/app/components/core/control-dispatcher/add-control-script/add-step/add-script/add-script.component.html [hunk 14]
```diff
@@ -37,7 +37,7 @@
 <ng-container *ngIf="scripts.length > 0">
   <div class="field is-horizontal">
     <div class="field-label is-small">
-      <label class="label">Execution</label>
+      <label class="label lbl-child">Execution</label>
```
Comment (praveen-garg) [inline src/app/components/core/control-dispatcher/add-control-script/add-step/add-script/add-script.component.html:40]: this is a child label?

File: src/app/components/core/control-dispatcher/add-control-script/add-step/add-script/add-script.component.html (hunk 13 above)
Comment (monika-Dianomic) [reply src/app/components/core/control-dispatcher/add-control-script/add-step/add-script/add-script.component.html:3]: yes

File: src/app/components/core/control-dispatcher/add-control-script/add-step/add-script/add-script.component.html (hunk 14 above)
Comment (monika-Dianomic) [reply src/app/components/core/control-dispatcher/add-control-script/add-step/add-script/add-script.component.html:40]: yes

File: src/styles.css [hunk 15]
```diff
@@ -418,4 +413,13 @@ a.dropdown-item:hover {
   .desc-text{
     font-size: 0.95rem;
   }
+
+  label {
```
Comment (praveen-garg) [inline src/styles.css:417]: If bulma makes it required that we always need class label with label tag; and has some CSS applied better to have it on .label, over-ridden

File: src/app/components/core/control-dispatcher/api-flow/add-edit-api-flow/add-edit-api-flow.component.html [hunk 16]
```diff
@@ -140,7 +140,7 @@
                                     <div *ngIf="af.destination && !((af.destination | titlecase) === 'Broadcast')"
                                     class="field is-horizontal mb-0">
                                     <div class="field-label is-small">
-                                        <label class="label is-small has-text-left">Name</label>
+                                        <label class="label lbl-child has-text-left">Name</label>
```
[the remark this answers is not in the document]
Comment (monika-Dianomic) [reply src/app/components/core/control-dispatcher/api-flow/add-edit-api-flow/add-edit-api-flow.component.html:143]: yes

File: src/app/components/core/control-dispatcher/pipelines/add-pipeline/add-control-pipeline.component.html (hunk 2 above)
Comment (monika-Dianomic) [reply src/app/components/core/control-dispatcher/pipelines/add-pipeline/add-control-pipeline.component.html:224]: You mean here, for child label I should not use `lbl-child` class i.e. `font-size: 0.85rem` ?

File: src/app/components/core/configuration-manager/configuration-group/configuration-group.component.css [hunk 17]
```diff
@@ -82,3 +82,8 @@
 .invalid-tab, .tabs.is-boxed li.is-active a.invalid-tab {
   border-bottom-color: #f14668 !important;
 }
+
+.is-small-font {
```
[the remark this answers is not in the document]
Comment (monika-Dianomic) [reply src/app/components/core/configuration-manager/configuration-group/configuration-group.component.css:86]: In that case other properties of label class/tag will also be applied to it.

File: src/app/components/core/logs/system-log/system-log.component.html [hunk 18]
```diff
@@ -19,7 +19,7 @@
       <div class="content mb-3">
         <div class="columns">
           <div class="column">
-            <label class="label is-small is-pulled-left">Service</label>
+            <label class="label is-size-7 is-pulled-left">Service</label>
```
Comment (mshariq-nerd) [inline src/app/components/core/logs/system-log/system-log.component.html:22]: we can remove `is-size-7`? the size is already given to the label.

File: src/app/components/core/logs/system-log/system-log.component.html [hunk 19]
```diff
@@ -54,7 +54,7 @@
             </div>
           </div>
           <div class="column">
-            <label class="label is-small">Severity</label>
+            <label class="label is-size-7">Severity</label>
```
Comment (mshariq-nerd) [inline src/app/components/core/logs/system-log/system-log.component.html:57]: As above

File: src/app/components/core/logs/system-log/system-log.component.html [hunk 20]
```diff
@@ -84,7 +84,7 @@
             </div>
           </div>
           <div class="column">
-            <label class="label is-small">Search</label>
+            <label class="label is-size-7">Search</label>
```
Comment (mshariq-nerd) [inline src/app/components/core/logs/system-log/system-log.component.html:87]: As above

File: src/app/components/core/settings/settings.component.html [hunk 21]
```diff
@@ -39,11 +39,11 @@
           </div>
         </div>
         <div class="column">
-          <label class="label is-small is-fullwidth">Host</label>
+          <label class="label is-size-7 is-fullwidth">Host</label>
```
Comment (mshariq-nerd) [inline src/app/components/core/settings/settings.component.html:42]: No need of `is-size-7` class. custom size of label is applied.

File: src/app/components/core/logs/audit-log/audit-log.component.html [hunk 22]
```diff
@@ -18,7 +18,7 @@
       <div class="content">
         <div class="columns is-mobile">
           <div class="column">
-            <label class="label is-small">Source</label>
+            <label class="label is-size-7">Source</label>
```
Comment (mshariq-nerd) [inline src/app/components/core/logs/audit-log/audit-log.component.html:21]: we can remove is-size-7. the size is already given to the label.

File: src/app/components/core/logs/audit-log/audit-log.component.html [hunk 23]
```diff
@@ -42,7 +42,7 @@
             </div>
           </div>
           <div class="column">
-            <label class="label is-small">Severity</label>
+            <label class="label is-size-7">Severity</label>
```
Comment (mshariq-nerd) [inline src/app/components/core/logs/audit-log/audit-log.component.html:45]: as above

File: src/app/components/core/logs/notification-log/notification-log.component.html [hunk 24]
```diff
@@ -18,7 +18,7 @@
       <div class="content">
         <div class="columns">
           <div class="column">
-            <label class="label is-small">Source</label>
+            <label class="label is-size-7">Source</label>
```
Comment (mshariq-nerd) [inline src/app/components/core/logs/notification-log/notification-log.component.html:21]: we can remove `is-size-7` class. the size is already given to the label.

File: src/app/components/core/logs/notification-log/notification-log.component.html [hunk 25]
```diff
@@ -42,7 +42,7 @@
             </div>
           </div>
           <div class="column">
-            <label class="label is-small">Severity</label>
+            <label class="label is-size-7">Severity</label>
```
Comment (mshariq-nerd) [inline src/app/components/core/logs/notification-log/notification-log.component.html:45]: as above

File: src/app/components/core/logs/notification-log/notification-log.component.html [hunk 26]
```diff
@@ -64,7 +64,7 @@
             </div>
           </div>
           <div class="column">
-            <label class="label is-small">Search</label>
+            <label class="label is-size-7">Search</label>
```
Comment (mshariq-nerd) [inline src/app/components/core/logs/notification-log/notification-log.component.html:67]: as above

File: src/app/components/core/settings/settings.component.html [hunk 27]
```diff
@@ -39,11 +39,11 @@
...
           <input id="host" class="input" type="text" placeholder="127.0.0.1" [ngModel]="host" trim="blur">
         </div>
         <div class="column">
-          <label class="label is-small is-fullwidth">Port</label>
+          <label class="label is-size-7 is-fullwidth">Port</label>
```
Comment (mshariq-nerd) [inline src/app/components/core/settings/settings.component.html:46]: as above

File: src/app/components/core/settings/settings.component.html [hunk 28]
```diff
@@ -78,7 +78,7 @@
     <div id="settings-content" class="card-content">
       <div class="columns">
         <div class="column is-3">
-          <label class="label is-small">Ping/Auto Refresh Interval</label>
+          <label class="label is-size-7">Ping/Auto Refresh Interval</label>
```
Comment (mshariq-nerd) [inline src/app/components/core/settings/settings.component.html:81]: as above

File: src/app/components/core/settings/settings.component.html [hunk 29]
```diff
@@ -109,7 +109,7 @@
         </div>
         <div class="column is-2"></div>
         <div class="column is-3">
-          <label class="label is-small">Timestamps</label>
+          <label class="label is-size-7">Timestamps</label>
```
Comment (mshariq-nerd) [inline src/app/components/core/settings/settings.component.html:112]: as above

File: src/app/components/core/settings/settings.component.html [hunk 30]
```diff
@@ -135,7 +135,7 @@
       </div>
       <div class="columns">
         <div class="column is-3">
-          <label class="label is-small"> Refresh Dashboard Graph</label>
+          <label class="label is-size-7"> Refresh Dashboard Graph</label>
```
Comment (mshariq-nerd) [inline src/app/components/core/settings/settings.component.html:138]: as above

File: src/app/components/core/settings/settings.component.html [hunk 31]
```diff
@@ -167,7 +167,7 @@
         </div>
         <div class="column is-2"></div>
         <div class="column is-3">
-          <label class="label is-small">Editor Theme</label>
+          <label class="label is-size-7">Editor Theme</label>
```
Comment (mshariq-nerd) [inline src/app/components/core/settings/settings.component.html:170]: as above

File: src/app/components/core/settings/settings.component.html [hunk 32]
```diff
@@ -193,7 +193,7 @@
 
       <div class="columns">
         <div class="column is-3">
-          <label class="label is-small">Readings Graph Default Duration</label>
+          <label class="label is-size-7">Readings Graph Default Duration</label>
```
Comment (mshariq-nerd) [inline src/app/components/core/settings/settings.component.html:196]: as above

File: src/app/components/core/settings/settings.component.html [hunk 33]
```diff
@@ -223,15 +223,15 @@
       <hr>
       <div class="columns">
         <div class="column is-3" *ngIf="rolesService.hasEditPermissions()">
-          <label class="label is-small">Developer Features</label>
+          <label class="label is-size-7">Developer Features</label>
```
Comment (mshariq-nerd) [inline src/app/components/core/settings/settings.component.html:226]: as above

File: src/app/components/core/settings/settings.component.html [hunk 34]
```diff
@@ -223,15 +223,15 @@
...
           <div class="control">
             <input type="checkbox" (click)="setDeveloperFeatures($event.target.checked)"
               [checked]="developerFeaturesService.getDeveloperFeatureControl()">
           </div>
         </div>
         <div class="column is-2"></div>
         <div class="column is-3">
-          <label class="label is-small">Readings Graph Alpha Control</label>
+          <label class="label is-size-7">Readings Graph Alpha Control</label>
```
Comment (mshariq-nerd) [inline src/app/components/core/settings/settings.component.html:234]: as above

**Review by mshariq-nerd — DISMISSED** [review]

**Review by hemantgarg923 — DISMISSED** [review]

**Review by hemantgarg923 — APPROVED** [review]

**Review by mshariq-nerd — APPROVED** [review]

**Review by praveen-garg — APPROVED** [review]

**Pull request merged**